src/classes/PMT_Payment_TDTM.cls

```diff
@@ -212,7 +267,7 @@ public class PMT_Payment_TDTM extends TDTM_Runnable {
             oppIds.add(pmt.npe01__Opportunity__c);
         }
 
-        Map<Id, Opportunity> oppsById = getPaymentOpportunities(oppIds);
+        oppsById = getPaymentOpportunities(oppIds);
```

Comment:
Make sure we call out that they need to retest Multicurrency functionality, given that we changed (albeit minor) stuff here.